.ci/magician/cmd/test_terraform_vcr.go

```diff
@@ -472,21 +505,21 @@ func formatComment(fileName string, tmplText string, data any) (string, error) {
 }
 
 func formatTestsAnalytics(data analytics) (string, error) {
-	return formatComment("test_terraform_vcr_test_analytics.tmpl", testsAnalyticsTmplText, data)
+	return formatComment("test_analytics.tmpl", testsAnalyticsTmplText, data)
 }
 
 func formatNonExercisedTests(data nonExercisedTests) (string, error) {
-	return formatComment("test_terraform_vcr_recording_mode_results.tmpl", nonExercisedTestsTmplText, data)
+	return formatComment("recording_mode_results.tmpl", nonExercisedTestsTmplText, data)
```

Comment:
nit: This isn't actually "used" but to be accurate... (it was already wrong in main.)

```suggestion
	return formatComment("non_exercised_tests.tmpl", nonExercisedTestsTmplText, data)
```